### ISSUE TITLE
University can't get past Level 1 - repeats Lvl2 at 100%

### ISSUE THREAD

**Issue by smmmadden** [ldtteam/minecolonies#6882]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [ x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 36.1.4
- MineColonies Version: 0.14.92-ALPHA
- Structurize Version: 0.13.163-ALPHA
- Related Mods and their Versions:

## Expected behavior

University showed as 100% in Builders Hut then move to next level.
## Actual behavior

University reaches 100% (but markers are still present) and then it reset back to 0% completed for Level 2 instead of moving forward.  Then the worker asks for all the same materials to not do anything because it was already done.  Still going through the same motions like its on some timer and not based on actual completion.

![image](https://user-images.githubusercontent.com/10309346/113961878-bcc87400-97f4-11eb-9ab1-5dfcaacb98da.png)
![image](https://user-images.githubusercontent.com/10309346/113961894-c5b94580-97f4-11eb-941f-f3229df9e5d4.png)
![image](https://user-images.githubusercontent.com/10309346/113962689-30b74c00-97f6-11eb-8a19-21b594229a6b.png)
The above shows a lot of blocks requested, but nothing happens with them.  The builder should be able to tell if the structure is completed for that level assuming there is a variable holding the current status/% completed and not waste time going through the motions for nothing. :-)
## Steps to reproduce the issue
Logs are still showing a number of exceptions with this build as well. 

Here is the lastest.log file.  It has university2 in it and the INFO message makes no sense:
[07Apr2021 22:12:48.117] [Server thread/INFO] [minecolonies/]: WorkOrderBuild at location BlockPos{x=-1126, y=110, z=73} is using schematics/spacewars/university2 instead of schematics/spacewars/university2
See the same messages for other structures as well.
1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by someaddons** [comment]
your laste.log file attachment seems to not have work, please reupload on https://gist.github.com/ and link here

**Comment by smmmadden** [comment]
sorry, thought I pasted it in.  Here it is: https://pastebin.com/tgznd048
I use PasteBin as I have a Pro Paid account and set to never expire.

**Comment by smmmadden** [comment]
looks like the builder finally got University Level 2 completed after many hours given the size of the build.  I verified that the number of citizens increased from 25 to 50, so we appear to be unblocked.  Don't know why it worked the second time either but it was a long wait to get there.  This should be able to be closed, though it does suggest that one builder for a large build like these, would be better with multiple builders like in real life, no one person could build a house by themselves. :-)

**Comment by Raycoms** [comment]
Is there a good way to reproduce this?
